### PR TITLE
plugin galactic.capital.investment

### PR DESCRIPTION
**Pull request by zuckung**
I think that should work.

### CODE REVIEW

**Review by MCOfficer — CHANGES_REQUESTED** [review]
Your tag is named `galactic.capital.investment.v1.1`, so this won't work. Either rename the tag to `v1.1` and remove the `v` from the autoupdate url, or change the version to `galactic.capital.investment.v1.1` and change the autoupdate url & iconUrl accordingly.

File: manifests/galactic.capital.investment.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: galactic.capital.investment 
+authors: zuckung
+homepage: https://github.com/zuckung/endless-sky-plugins/
+license: MIT
+version: v1.1
```
Comment (MCOfficer) [inline manifests/galactic.capital.investment.yaml:5]:
```suggestion
version: v1.1-galactic.capital.investment
```
analogous to your tag name - it's a bit ugly, but works

**Comment by zuckung** [comment]
The workflows say something about a linebreak and syntax

File: manifests/galactic.capital.investment.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: galactic.capital.investment
```
Comment (MCOfficer) [inline manifests/galactic.capital.investment.yaml:1]:
```suggestion
name: galactic.capital.investment
```

File: manifests/galactic.capital.investment.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: galactic.capital.investment 
+authors: zuckung
+homepage: https://github.com/zuckung/endless-sky-plugins/
+license: MIT
+version: v1.1-galactic.capital.investment
+shortDescription: Implements a short mission chain that enables repeatable job board investment opportunities which result in small daily income. Available in human, quarg and hai space.
+description: >- Implements a short mission chain that enables repeatable job board investment opportunities which result in small daily income. Available in human, quarg and hai space starting with 2 million credits cash and going up to 1000 million credits. See the readme for details.
```
Comment (MCOfficer) [inline manifests/galactic.capital.investment.yaml:7]:
```suggestion
description: >-
  Implements a short mission chain that enables repeatable job board investment opportunities which result in small daily income. Available in human, quarg and hai space starting with 2 million credits cash and going up to 1000 million credits. See the readme for details.
```
yaml multiline syntax is a mess... this should work

**Comment by zuckung** [comment]
Yay, thx for your time ;)

**Pull request merged**